src/Api/appsettings.Production.json

```diff
@@ -22,5 +22,18 @@
     "bitPay": {
       "production": true
     }
+  },
+  "Quartz": {
+    "quartz.scheduler.instanceId": "AUTO",
+
+    "quartz.jobStore.type": "Quartz.Impl.AdoJobStore.JobStoreTX",
+    "quartz.jobStore.driverDelegateType": "Quartz.Impl.AdoJobStore.SqlServerDelegate",
+    "quartz.jobStore.useProperties": true,
+    "quartz.jobStore.dataSource": "myDS",
+    "quartz.jobStore.tablePrefix": "QRTZ_",
+
+    "quartz.jobStore.clustered": true,
+
+    "quartz.dataSource.myDS.provider": "SqlServer"
   }
```

Comment:
We should configure in code and only pull the specific things users need to configure (if any) from appsettings.